querydsl-sql-codegen/src/main/java/com/querydsl/sql/codegen/MetaDataExporter.java

```diff
@@ -372,15 +387,19 @@ private void serialize(EntityType type) {
         try {
             String fileSuffix = createScalaSources ? ".scala" : ".java";
 
+            String schemaName = (String) type.getData().get("schema");
+            String tableName = (String) type.getData().get("table");
+            SchemaAndTable schemaAndTable = new SchemaAndTable(schemaName, tableName);
```

Comment:
the SchemaAndTable instance which is created in the previous could be given as an argument, then the normalized schema and table name would be used in both cases.